tempesta_fw/http_parser.c

```diff
@@ -2108,7 +2072,7 @@ __resp_parse_expires(TfwHttpResp *resp, unsigned char *data, size_t len)
 		__fsm_sz = len - (size_t)(p - data);
 		__fsm_n = parse_int_a(p, __fsm_sz, colon_a, &parser->_tmp_acc);
 		if (__fsm_n == CSTR_POSTPONE)
-			tfw_http_msg_hdr_chunk_fixup(msg, p, __fsm_sz);
+			tfw_http_msg_field_chunk_fixup(msg, &msg->parser.hdr, p, __fsm_sz);
```

Comment:
Too long lines with the new function call. You can write tfw_http_msg_hdr_chunk_fixup(msg, data, size) which just calls tfw_http_msg_field_chunk_fixup(msg, &msg->parser.hdr, data, size) to avoid many calls of the new function with many arguments.